### PR TITLE
Automated tests for Vehicle App Cpp template

### PR DESCRIPTION
**Pull request by mml5bg**
# Description

## Checklist

* [x] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [x] Vehicle App can process MQTT messages and call the seat service
* [x] Vehicle App can be deployed to local K3D and is running
* [x] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [x] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: sdk
Comment (MP91) [inline sdk:1]: we don't need that anymore, do we?

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ conan==1.60.2
 
 pre-commit==2.20.0
 cpplint==1.6.1
+pytest==7.4.1
```
Comment (MP91) [inline requirements.txt]: do we really need this in the main requirements.txt?

File: .devcontainer/tests/automated_tests/runtime_test.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    subprocess.check_output(["velocitas", "package", "-p", "devenv-runtimes"])  # nosec
+    .decode("utf-8")
+    .strip("\n")
+)
+
+os.environ["VDB_PORT"] = "30555"
+os.environ["MQTT_PORT"] = "31883"
+
+
+class RuntimeTest(unittest.TestCase):
```
Comment (MP91) [inline .devcontainer/tests/automated_tests/runtime_test.py:31]: should we also add a test for the conan requirements?

**Review by MP91 — CHANGES_REQUESTED** [review]

File: .devcontainer/tests/automated_tests/runtime_test.py (hunk 2 above)
Comment (doosuu) [reply .devcontainer/tests/automated_tests/runtime_test.py:31]: Not easily. I suggest we add a test like that later (if at all)

File: sdk
Comment (doosuu) [reply sdk:1]: You are right. Snuck itself in.

File: requirements.txt (hunk 1 above)
Comment (doosuu) [reply requirements.txt]: Moved to test requirements.

**Review by MP91 — APPROVED** [review]
lgtm

**Pull request merged**